scalalib/src/mill/scalalib/TestModule.scala

```diff
@@ -423,17 +431,17 @@ object TestModule {
         )
     ).getOrElse("")
     e.status match {
-      case Error if (e.exceptionMsg.isDefined && e.exceptionName.isDefined) =>
+      case ErrorStatus if (e.exceptionMsg.isDefined && e.exceptionName.isDefined) =>
```

Comment:
That's an interesting change. The old type must have been some kind of `Status` or `enum`, the new type is `String`. So either the old or the new version doesn't work or I'm missing something.

Reply:
Nevermind. Just realized, the renamed the `val`s.